### PR TITLE
chore: docs cleanup oct 22

### PR DESCRIPTION
**Pull request by thorwebdev**
- Update example links
- Pull out duplicate content in quickstarts (intro; project setup steps) into components

@dannykng could I get your eyes on the components set up? e.g. is `guides/_components` a good place for these?

### CODE REVIEW

File: apps/reference/nav/_referenceSidebars.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const sidebars = {
       type: 'category',
       label: 'Overview',
       collapsed: true,
-      items: ['about', 'guides/features', 'architecture', 'guides/examples'],
```
Comment (kiwicopple) [inline apps/reference/nav/_referenceSidebars.js:7]: is this correct? We don't want the examples page anymore?

Comment (thorwebdev) [reply apps/reference/nav/_referenceSidebars.js:7]: I think in its current state it's more overwhelming rather than useful. Wdyt?

**Comment by dannykng** [comment]
> dannykng could I get your eyes on the components set up? e.g. is `guides/_components` a good place for these?

LGTM but should we do `guides/_partials` instead so there's no confusion about it being a directory of React components?

File: apps/reference/nav/_referenceSidebars.js (hunk 1 above)
Comment (dannykng) [reply apps/reference/nav/_referenceSidebars.js:7]: I remember feeling the same when I first came across it. Do we need to add a redirect for it or just taking it out of the sidebar temporarily?

File: apps/reference/docs/guides/_components/project_setup.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+import UserManagementSQLTemplate from './user_management_quickstart_sql_template.md'
+
+## Project set up
```
Comment (dannykng) [inline apps/reference/docs/guides/_components/project_setup.mdx:5]: Hmm looks like the table of contents isn't playing nicely with these partials/components:
Screen Shot 2022-10-20 at 11 45 46 AM

What headings should be included:
Screen Shot 2022-10-20 at 11 47 04 AM

Comment (thorwebdev) [reply apps/reference/docs/guides/_components/project_setup.mdx:5]: @dannykng any idea how to fix that? I'd prefer not to pull out the headlines. I'd say let's not block on this and fix in a separate PR!

Comment (kiwicopple) [reply apps/reference/docs/guides/_components/project_setup.mdx]: the only way to fix it is to keep the Headings in the page, and then just the content in the partial

I think the partials also mess up the navigation a bit (eg, you can "navigate" to the partial like it's an actual doc)

**Review by kiwicopple — APPROVED** [review]
yup, I think it's good for now. Pretty urgent that we get this one out 👍 

Thanks for all the hard work here @thorwebdev

**Pull request merged**